corehq/apps/users/models.py

```diff
@@ -2124,6 +2124,18 @@ def get_sql_location(self, domain):
         if loc_id:
             return SQLLocation.objects.get_or_None(domain=domain, location_id=loc_id)
 
+    def get_assigned_location_ids(self, domain):
+        return getattr(self.get_domain_membership(domain), 'assigned_location_ids', None)
+
+    @memoized
+    def get_assigned_sql_locations(self, domain):
+        from corehq.apps.locations.models import SQLLocation
+        loc_ids = self.get_assigned_location_ids(domain)
+        if loc_ids:
+            return SQLLocation.objects.get_locations(loc_ids)
+        else:
+            return []
+
```

Comment:
Yeah, it sounds fine to convert mine over (it wouldn't make sense to have both versions of it) as long as CommCareUser and WebUser both have the same method names and arguments.  Apparently CommCareUsers sometimes make web requests, and I've seen some issues where we expect a web user, so we call a WebUser method, but that breaks if it's a CommCareUser.